runtime/oti/xhelpers.m4

```diff
@@ -79,10 +81,12 @@ define({FILE_START},{
 
 define({C_FUNCTION_SYMBOL},_$1)
 
+define({GLOBAL_SYMBOL},_$1)
+
 define({START_PROC},{
```

Comment:
The two copies of START_PROC (osx and linux) are now identical and can be moved out of the ifdef